tests/plugins/editorplaceholder/editorplaceholder.js

```diff
@@ -205,5 +219,101 @@
 		} );
 	};
 
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin' ] = function() {
+		consoleErrorStub = sinon.stub( console, 'error' );
+
+		var runtimeErrorListener = function() {
+			resume( function() {
+				assert.fail( 'There should be no RunTime errors!' );
+			} );
+		};
+
+		window.addEventListener( 'error', runtimeErrorListener );
+
+		bender.editorBot.create( {
+			name: 'bbcodeFailTest',
+			config: {
+				extraPlugins: 'bbcode',
+				editorplaceholder: 'any',
+				fullPage: true,
+				on: {
+					instanceReady: function() {
+						resume( function() {
+							assert.isFalse( consoleErrorStub.called, 'There should be no errors during initialization' );
+						} );
+					}
+				}
+			}
+		}, function( bot ) {
+
+		} );
+
+		wait();
+	};
+
+	// (#4253)
+	tests[ 'test placeholder loads correctly in full-page editor with bbcode plugin and initial content' ] = function() {
+		bender.editorBot.create( {
```

Comment:
Added log error checking for another test with pattern:
```
addErrorAsserts();

//editor creation

wait()
```
Works fine for `test placeholder loads correctly in full-page editor with bbcode plugin and initial content`.

Try the same with `test placeholder not visible with data filter to preinitialized content`. This time error is not handled.
But if I add `extraPlugins: 'bbcode'` - it works fine... The error is caught and test fail due to RuntimeError...  :thinking: 

With `bbcode` this error looks like error in console. But without this plugin it looks like casual `log`

![image](https://user-images.githubusercontent.com/6498567/99593542-1b31b900-29f2-11eb-84a7-fccfb067495b.png)
Tried with another extraPlugins like: `htmlwriter`, `easyimage`, `devtools`, `image2` and `mathjax` - but non of them behaved like `bbcode`. So adding each as the only extraPlugin gives same result as without any plugin for this test.

Reply:
Finally, after talk with @f1ames there will be only one test for RunTime error check. Since it's the core sense of this issue.
The rest of them just check edge cases in data.